src/flutter/shell/platform/linux_embedded/window/linuxes_window_drm.cc

```diff
@@ -97,7 +97,16 @@ bool LinuxesWindowDrm::DispatchEvent() {
 }
 
 bool LinuxesWindowDrm::CreateRenderSurface(int32_t width, int32_t height) {
-  native_window_ = std::make_unique<NativeWindowDrm>(kDrmDeviceFilename);
+  auto device_filename = std::getenv(kFlutterDrmDevice);
+  if ((!device_filename) || (device_filename[0] == '\0')) {
+    LINUXES_LOG(WARNING) << kFlutterDrmDevice << " is not set, use "
+                         << kDrmDeviceDefaultFilename;
+    native_window_ =
+        std::make_unique<NativeWindowDrm>(kDrmDeviceDefaultFilename);
```

Comment:
Why do you call the same API in if-else? You must not use if-else.

Reply:
I fixed it.